Source/Scene/Scene.js

```diff
@@ -1752,9 +1801,16 @@ define([
 
         this._tweens.update();
         this._camera.update(this._mode);
-        this._screenSpaceCameraController.update();
+
+        if (!this._useWebVR || this._mode === SceneMode.SCENE2D) {
```

Comment:
Also, does it make sense to always disable the screen-space camera controller?  Or is that a app decision?  Won't it differ on desktop vs. mobile?